scala-thoughts.md

```diff
@@ -0,0 +1,15 @@
+### What's easy to do in Scala? What's not?
+There are built-in functions that make pattern-matching recursion seem extraordinarily simple.
+Well, I would say that it is conceptually dense, like Racket. The amount of thought that each line takes is quite a bit, though I may be biased since this is the first time I've used Scala.
+I would say it is not that difficult to do anything, at least compared to other general purpose programming languages. I feel like it is powerful in what I can accomplish in a few lines of code, but at the same time has the ability to take on any task -- I can definitely notice the Java peeking through. 
```

Comment:
I'm curious about how you see Java emerging. What parts of the language feel like Java?